### PR TITLE
Article Menu Item

### PR DESCRIPTION
**Pull request by samarthsharma36**
You also need MenuItemStep file which is another PR of Menu Item #43 .
Please review and suggest changes @puneet0191  @yvesh  @nhungredweb

### CODE REVIEW

File: src/_support/Page/Acceptance/Administrator/ContentListPage.php [hunk 1]
```diff
@@ -14,71 +14,99 @@
...
-	 * Locator for article's name field
+	 * Locator for article's featured icon
 	 *
-	 * @var    array
-	 * @since  __DEPLOY_VERSION__
+	 * @var    string
+	 * @since  4.0.0
 	 */
-	public static $seeName = ['xpath' => "//table[@id='articleList']//tr[1]//td[4]"];
+	public static $seeFeatured = "//table[@id='articleList']//*//span[@class='icon-featured']";
```
Comment (puneet0191) [inline src/_support/Page/Acceptance/Administrator/ContentListPage.php]: use of wildcard in xpath is going to be very costly, try to be absolutely sure about the exact path please.

File: src/acceptance/administrator/components/com_menu_article/ArticleMenuCest.php [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+		// Assertion
+		$I->setFilter('select status', 'Published');
+		$I->searchForItem($this->articleTitle);
+		$I->see($this->articleTitle);
+		// FrontEnd Assertion
+		FrontEnd::articleIsVisible($I, $this->menuItemName, $this->articleTitle);
+	}
+
+	/**
+	 * Trah Article
```
Comment (puneet0191) [inline src/acceptance/administrator/components/com_menu_article/ArticleMenuCest.php]: Trash*

File: src/acceptance/administrator/components/com_menu_article/ArticleMenuCest.php [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+		$I->amOnPage(ContentListPage::$url);
+		$I->checkForPhpNoticesOrWarnings();
+		// Trash
+		$I->trashArticle($this->articleTitle);
+		// Backend Assertion
+		$I->setFilter('select status', 'Trashed');
+		$I->searchForItem($this->articleTitle);
+		$I->see($this->articleTitle);
+		// FrontEnd Assertion
+		FrontEnd::articleIsNotVisible($I, $this->menuItemName, $this->articleTitle);
```
Comment (puneet0191) [inline src/acceptance/administrator/components/com_menu_article/ArticleMenuCest.php]: you could just inject the step object to this test

**Review by puneet0191 — CHANGES_REQUESTED** [review]
Minor changes please

**Comment by samarthsharma36** [comment]
@puneet0191 @yvesh @nhungredweb Please review again. I made the changes

**Review by puneet0191 — APPROVED** [review]

**Pull request merged**